### DISCUSSION TITLE
Text for C-API.

### DISCUSSION THREAD

**Discussion by Jean-Amaro** [thorvg/thorvg#2340] [Q&A] [answered]
Is there a way to create a text, using the C-API, similarly to what `tvg::Text` does in C++?

**Comment by hermet** [comment]
@Jean-Amaro Hello, just the APIs are missing. We will take over soon. Please track the issue here: https://github.com/thorvg/thorvg/issues/2374

**Comment by hermet** [comment, marked as answer]
@Jean-Amaro Please have a look at the API prototypes the ThorVG developers are pushing, and let us know if you have any concerns in advance. Thanks.
```
TVG_API Tvg_Paint* tvg_text_new(void);
TVG_API Tvg_Result tvg_text_set_font(Tvg_Paint* paint, const char* name, float size, const char* style);
TVG_API Tvg_Result tvg_text_set_text(Tvg_Paint* paint, const char* text);
TVG_API Tvg_Result tvg_text_set_fill_color(Tvg_Paint* paint, uint8_t r, uint8_t g, uint8_t b);
TVG_API Tvg_Result tvg_text_set_linear_gradient(Tvg_Paint* paint, Tvg_Gradient* gradient);
TVG_API Tvg_Result tvg_text_set_radial_gradient(Tvg_Paint* paint, Tvg_Gradient* gradient);
TVG_API Tvg_Result tvg_load_font(const char* path);
TVG_API Tvg_Result tvg_unload_font(const char* path);
```
\+ @mgrudzinska

**Reply by Jean-Amaro** [reply]
Hello!
The proposed prototypes are great.
Thanks, @hermet and @mgrudzinska!

**Comment by hermet** [comment]
@Jean-Amaro hello, the text capi will be supported in v0.14. thanks.  
=> https://github.com/thorvg/thorvg/pull/2377/commits/92a5bb9eaa90c27b9d7256eb3824b8c2cb50c37d
